Documentation/MSTest-Migration-EditorConfig.txt

```diff
@@ -0,0 +1,125 @@
+dotnet_diagnostic.MSTEST0048.severity = warning  # TestContext property usage
+dotnet_diagnostic.MSTEST0049.severity = suggestion # Flow TestContext cancellation token
+dotnet_diagnostic.MSTEST0050.severity = error    # Global test fixture should be valid
+
+# ==================================================
+# PHILIPS MSTEST ANALYZERS (Migration Settings)
+# ==================================================
+
+# DISABLE - Rules with direct Microsoft equivalents
+dotnet_diagnostic.PH2004.severity = none  # Use MSTEST0006 instead (ExpectedException)
+dotnet_diagnostic.PH2005.severity = none  # Use MSTEST0005 instead (TestContext usage)
+dotnet_diagnostic.PH2013.severity = none  # Use MSTEST0015 instead (Ignore attribute)
+dotnet_diagnostic.PH2016.severity = none  # Use MSTEST0008 instead (TestInitialize)
+dotnet_diagnostic.PH2017.severity = none  # Use MSTEST0010 instead (ClassInitialize)
+dotnet_diagnostic.PH2018.severity = none  # Use MSTEST0011 instead (ClassCleanup)
+dotnet_diagnostic.PH2019.severity = none  # Use MSTEST0009 instead (TestCleanup)
+dotnet_diagnostic.PH2033.severity = none  # Use MSTEST0014 instead (DataTestMethod requires DataRows)
+dotnet_diagnostic.PH2034.severity = none  # Use MSTEST0030 instead (TestMethod requires TestClass)
+dotnet_diagnostic.PH2036.severity = none  # Use MSTEST0003 instead (TestMethod must be public)
+dotnet_diagnostic.PH2038.severity = none  # Use MSTEST0002 instead (TestClass must be public)
+dotnet_diagnostic.PH2058.severity = none  # Use MSTEST0026 instead (Avoid Assert conditional check)
+dotnet_diagnostic.PH2059.severity = none  # Use MSTEST0029 instead (Public methods should be TestMethod)
+
+# CONSIDER DISABLING - Rules with similar Microsoft functionality
+dotnet_diagnostic.PH2003.severity = none  # Consider MSTEST0037 instead (Assert.AreEqual usage)
+dotnet_diagnostic.PH2008.severity = none  # Consider MSTEST0037 instead (Assert parameter types)
+dotnet_diagnostic.PH2009.severity = none  # Consider MSTEST0037 instead (IsTrue/IsFalse usage)
+dotnet_diagnostic.PH2035.severity = none  # Consider MSTEST0014 instead (DataTestMethod parameter count)
+dotnet_diagnostic.PH2037.severity = none  # Consider MSTEST0003 instead (Unique test names)
+dotnet_diagnostic.PH2050.severity = none  # Consider MSTEST0003 instead (Empty TestMethod)
+dotnet_diagnostic.PH2055.severity = none  # Consider MSTEST0037 instead (Assert.IsTrue(true))
+dotnet_diagnostic.PH2056.severity = none  # Consider MSTEST0037 instead (Assert.AreEqual(true, true))
+dotnet_diagnostic.PH2076.severity = none  # Consider MSTEST0025 instead (Assert.Fail alternatives)
+dotnet_diagnostic.PH2095.severity = none  # Consider MSTEST0003 instead (TestMethod return types)
+
+# KEEP - Philips-specific rules that provide unique value
+dotnet_diagnostic.PH2000.severity = suggestion  # Test method naming convention (unique to Philips)
+dotnet_diagnostic.PH2010.severity = none       # Parentheses style (optional style preference)
```

Comment:
If you look in the PH2010.md file, there's a section for possible alternatives. (In this case, it's an IDE rule.) For ALL of these rules (where applicable), update the corresponding .md file to point to the MS equivalent. Also provide a link to the generic MSTest-Migration-Guide.md file within all these individual md files. (Why? because when a finding occurs in the IDE, that's the link the user will see - the individual rule md file. No one will ever find the MSTest-Migration-Guide.md file.)